osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -245,8 +246,9 @@ public void Download(BeatmapSetInfo beatmapSetInfo, bool noVideo = false)
                 return;
             }
 
-            ProgressNotification downloadNotification = new ProgressNotification
+            var downloadNotification = new ProgressNotification
             {
+                CompletionText = $"Installed {beatmapSetInfo.Metadata.Artist} - {beatmapSetInfo.Metadata.Title}!",

```

Comment:
How about "Imported ..."

Reply:
Sure, if that is the preferred way of describing it